From b4e094f2b93fb0b258aeb0be88f25cb2f3bdfde5 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@freescale.com>
Date: Wed, 12 Aug 2015 16:20:42 +0800
Subject: [PATCH 0174/5242] MLK-11352-1 ARM: dts: imx7d-12x12-lpddr3: add dts
 support for pxp-v4l2-output

commit  af12379954c7c6c57f93b86941388cfa54923e61 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add dts support for pxp-v4l2-output module for
imx7d-12x12-lpddr3 board.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
index bdf5597..08ffb26 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
@@ -22,6 +22,11 @@
 		status = "okay";
 	};
 
+	pxp_v4l2_out {
+		compatible = "fsl,imx7d-pxp-v4l2", "fsl,imx6sx-pxp-v4l2", "fsl,imx6sl-pxp-v4l2";
+		status = "okay";
+	};
+
 	regulators {
 		compatible = "simple-bus";
 		#address-cells = <1>;
-- 
1.7.9.5

